docs/installation/index.md

```diff
@@ -18,19 +18,36 @@ The Autoware Foundation provides no support on other platforms than those listed
 
 Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
 
-## Installation steps of Autoware
+## Autoware
```

Comment:
Hmm, I feel it seems a bit ambiguous with just "Autoware". :thinking:
@LalithVipulananthan @shmpwk Is it okay for you? 

```md
# Installation
## Target platforms
## Autoware
## Related tools
```

Reply:
The changing comes from here.
https://github.com/autowarefoundation/autoware-documentation/pull/104#discussion_r873280983

Looking at ROS2 documentation, "Installing hoge" might be easier to understand. :thinking:
![Screenshot from 2022-05-16 18-47-56](https://user-images.githubusercontent.com/42209144/168566111-af02b243-0d81-4cfe-9b0d-e6c1b347878f.png)

Reply:
Looks good!

Reply:
Fixed by  https://github.com/autowarefoundation/autoware-documentation/pull/104/commits/f8b939b625e3ea9d19ae5706298549cb78272082